src/fft-api/parcel/fftParcelService.ts

```diff
@@ -27,4 +28,23 @@ export class FftParcelService {
   public async findMultiple(ids: string[]): Promise<Parcel[]> {
     return await Promise.all(ids.map(async (i) => await this.findById(i)));
   }
+
+  public async getLabel(parcelId: string, labelDocument: string): Promise<Buffer> {
```

Comment:
hier fände ich einen enum für die erlaubten `labelDocument` Werte (all.pdf, send.pdf and return.pdf) einfacher zu benutzen